Gemfile

```diff
@@ -10,10 +10,3 @@ group :coverage, optional: ENV['COVERAGE']!='yes' do
   gem 'simplecov-console', :require => false
   gem 'codecov', :require => false
 end
-
-group :tests do
```

Comment:
Is this really a good thing? Looks like today we don't run Rubocop in the CI workflow, but shouldn't we?